shacl12-core/index.html

```diff
@@ -6413,6 +6411,98 @@ <h4>sh:property</h4>
+								<th>Summary and Syntax Rules</th>
+							</tr>
+						</thead>
+						<tbody>
+							<tr>
+								<td><code>sh:someValue</code></td>
+								<td>
+									The <a>shape</a> that at least one of the <a>value nodes</a> needs to conform to.
+									<span data-syntax-rule="someValue-shape">The values of <code>sh:someValue</code> in a shape must be <a>well-formed</a> <a>shapes</a>.</span>
+								</td>
+							</tr>
+						</tbody>
+					</table>
+					<div class="def def-text">
+						<div class="def-header">TEXTUAL DEFINITION</div>
+						<div class="def-text-body" data-validator="Node">
+							Let <code>$someValue</code> be a <a>value</a> of <code>sh:someValue</code>.
+							A <a>failure</a> MUST be produced if the <a>conformance checking</a> of any <a>value node</a> against
+							<code>$someValue</code> produces a <a>failure</a>, unless at least one of the <a>value nodes</a> conforms to <code>$someValue</code>.
+							Otherwise, if none of the <a>value nodes</a> <a>conforms</a> to <code>$someValue</code>,
+							there is a <a>validation result</a>.
+						</div>
+					</div>
+					<p><em>The remainder of this section is informative.</em></p>
+					<p>
+						Note that implementations may stop processing value nodes once one of the value nodes has conformed.
+						Since the order of nodes in the set of value nodes is undefined, there is no guarantee that any node is
+						reached that would cause a failure.
+						Therefore, to make processing predictable, failures are silently ignored unless all value nodes have been visited without success.
+					</p>
+					<p>
+						In the following example, any nodes that conform to <code>ex:DuckFarmerShape</code> need to have at least one value
+						for <code>ex:tendsAnimal</code> that is a <a>SHACL instance</a> of <code>ex:Duck</code>.
+					</p>
+					<aside class="example">
+						<div class="shapes-graph">
+							<div class="turtle">
+ex:DuckFarmerShape
+    a sh:NodeShape ;
+    sh:property [
+        sh:path ex:tendsAnimal ;
+        sh:someValue [
+            sh:class ex:Duck ;
+        ]	
+    ] .
+							</div>
+							<div class="jsonld">
+								<pre class="jsonld">{
+	"@graph": [
+		{
+			"@id": "ex:DuckFarmerShape",
+			"@type": "sh:NodeShape",
+			"sh:property": {
+				"sh:path": {
+					"@id": "ex:tendsAnimal"
+				},
+				"sh:someValue": {
+					"sh:class": {
+						"@id": "ex:Duck"
+					}
+				}
+			}
+		}
+	]
+}</pre>
+						</div>
+						<div class="shaclc"><pre class="shaclc">TODO</pre></div>
+					</aside>
+					<p>
+						<code>sh:someValue</code> can be regarded as syntactic sugar for a combination of
+						<code>sh:qualifiedValueShape</code> and <code>sh:qualifiedMinCount 1</code>.
+					</p>
+					<p>
+						Also note that <code>sh:someValue</code> mostly makes sense when used in <a>property shapes</a>.
+						In <a>node shapes</a>, <code>sh:someValue</code> is equivalent to using <code>sh:node</code> or <code>sh:property</code>.
```

Comment:
I think we need to dig into this a bit more semantically.  If I use `sh:someValue` on a node shape, could I be expressing that I just want there to be at least one conformant node in the graph?  "Equivalent to using `sh:node`" feels like a _universal_ constraint rather than an _existential_ constraint.

```turtle
ex:DuckFarmerShape2
    a sh:NodeShape ;
    sh:someValue [
        sh:property [
            sh:path ex:tendsAnimal ;
            sh:class ex:Duck ;
        ]
    ] .
```

vs.

```turtle
ex:DuckFarmerShape3
    a sh:NodeShape ;
    sh:node [
        sh:property [
            sh:path ex:tendsAnimal ;
            sh:class ex:Duck ;
        ]
    ] .
```

`ex:DuckFarmerShape2` and `ex:DuckFarmerShape3` are not equivalent.